_includes/tags.html

```diff
@@ -19,19 +19,19 @@
                        using the literal string '/blog/'. Path is used to
                        create the filtered URL: {{ path }}?filter_tags={{ tag }}.
 
-   display_header:     The optional boolean flag to display the tags_header element.
-                       Defaults to True. Adds class 'tags__hide-header' if falsy
-                       value passed (e.g., '', False, None).
+   hide_header:        The optional boolean flag to hide the tags_header element.
+                       Defaults to False. Adds class 'tags__hide-header' if True
+                       value passed (e.g., '', True, None).
```

Comment:
It doesn't equal True though, right? It's an absence of value I think? So if you pass None as a parameter would it fall back to the default (False)? I'm having trouble running sheer at the moment so can't test.